### PR TITLE
Clean IPFS URL usage and force user to provide an API link

### PR DESCRIPTION
**Pull request by dapplion**

### CODE REVIEW

File: src/params.ts [hunk 1]
```diff
@@ -81,7 +81,8 @@ export const defaultLocalAragonBases = {
 }
 
 export const defaultIpfsGateway = 'https://ipfs.eth.aragon.network'
-export const defaultIpfsApiUrl = 'http://localhost:5001'
+// No default ipfsApiUrl is provided. User must intentionally provide one
+// export const defaultIpfsApiUrl
```
Comment (0xGabi) [inline src/params.ts:85]: Why leave it commented?

Comment (dapplion) [reply src/params.ts:85]: So ourselves in the future don't add it again

**Comment by sohkai** [comment]
So something I just realized recently was pinata supports uploading an entire local directory and pinning it.

Not sure how easy it would be to do this, but I wonder if we can keep the temporary folder or move it somewhere that would be easy to upload? You can technically get the folder yourself through `ipfs get <cid>` as well.

**Comment by dapplion** [comment]
> So something I just realized recently was pinata supports uploading an entire local directory and pinning it.
> 
> Not sure how easy it would be to do this, but I wonder if we can keep the temporary folder or move it somewhere that would be easy to upload? You can technically get the folder yourself through `ipfs get <cid>` as well.

We don't use temporary directories while publishing with the plugin. The release directory (`dist` typically) is populated and persisted locally.

**Comment by sohkai** [comment]
Nice, then it might be as simple as just adding a log line!

Just wondering—it would be nice if publish detected that the `dist/` folder was already dirty and prompted the user to clean it. Should I lift this into a nice to have issue?

**Comment by dapplion** [comment]
> Nice, then it might be as simple as just adding a log line!
> 
> Just wondering—it would be nice if publish detected that the `dist/` folder was already dirty and prompted the user to clean it. Should I lift this into a nice to have issue?

Please! I was thinking that we should definitely clean `dist` to be sure of not introducing un-wanted artifacts

**Pull request merged**